### DISCUSSION TITLE
No stable firmware for pico W?

### DISCUSSION THREAD

**Discussion by gofschreiber** [micropython/micropython#10292] [RP2040 / Pico]
Hi, 
on https://micropython.org/download no stable firmware version is listed for .../rp2-pico-w. .../rp2-pico/ has some. 
- Is there a stable version for the pico W?
- Where can I find it?
- How can I make Thonny aware of it?
Thanks!

**Comment by robert-hh** [comment]
For Pico W there exist "only" daily builds, with all recent updates and bug fixes. It is safe to use it. There will be a "stable" build once V1.20 will be published.  With Thonny you should be able to select rp2040 as board type.

**Comment by gofschreiber** [comment]
Thanks for your quick answer!
I guess I got irritated by the "unstable" in the name of the image.

**Reply by scruss** [reply]
Stable [v1.19.1](https://github.com/micropython/micropython/tree/v1.19.1) was issued a couple of weeks before the Pico W was released on 2022-06-30. Nightly builds are *almost* always entirely safe to use. I now only use stable builds (when they're available) to see if a problem is really a bug in MicroPython or something in my code. So far, it's always been my code that was wrong.

**Comment by mattytrentini** [comment]
Stay tuned, rumour has it that v1.20 is _imminent_!
